pkg/util/finalizer_test.go

```diff
@@ -0,0 +1,47 @@
+package util
+
+import (
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubevirtv1 "kubevirt.io/api/core/v1"
+	"testing"
```

Comment:
"testing" should be at the top as a system package

Reply:
👍